server/reload.go

```diff
@@ -681,12 +681,31 @@ func (s *Server) applyOptions(opts []option) {
 	s.Noticef("Reloaded server configuration")
 }
 
+// On reload we have clients that may exists that are connected
+// but are not assigned to an account. Check here and assign to
+// the global account.
+// Lock should be held.
+func (s *Server) assignGlobalAccountToOrphanUsers() {
```

Comment:
Not sure this is the right thing to do. On startup, users without account have `u.Account == nil`. So either we are consistent and always assign the global account on parsing of users (without account), or we fix `clientHasMovedToDifferentAccount()` by setting the initial `curAccName` value to `globalAccountName` and also patch the check. For instance for users, it would be:
```
} else if u != nil {
		if u.Account != nil {
			return curAccName != u.Account.Name
		}
		// This is an user without an account. The client's has moved
		// account if its currant account name is not the global account name.
		return curAccName != globalAccountName
	}
```

Reply:
on first load it is never nil. We assign to global. But on reload we don't follow same path, you call configAuthorization inline, so we needed to add same logic here.